src/send_kcidb.py

```diff
@@ -46,6 +50,34 @@ def __call__(self, configs, args):
 
         sys.stdout.flush()
 
+    def get_revision(self, node):
+        return {
+            "builds": [],
+            "checkouts": [
+                {
+                    "id": node["_id"],
```

Comment:
The value of this field needs to start with `kernelci` as [described in the schema](https://github.com/kernelci/kcidb-io/blob/cd85bfed4eb3c1df8018b89aa7227c194a118ae5/kcidb_io/schema/v4.py#L130-L143).

Reply:
Okay